### ISSUE TITLE
how to convert these numbers into the real cell type ?

### ISSUE THREAD

**Issue by noob000007** [OmicsML/CellPLM#17]
run these code：
pipeline.predict(
                data, # An AnnData object
                pipeline_config, # The config dictionary we created previously, optional
            )
            
will get res like:tensor([13,  0, 13,  ...,  3,  8,  6], device='cuda:3')

so , how to convert these into the real cell type ?

**Comment by wehos** [comment]
Hi there,

There was a dictionary of label encoders here in the dataset module below. You should be able to find the corresponding label encoder in this dictionary, which is an object of sklearn label encoder (`https://scikit-learn.org/dev/modules/generated/sklearn.preprocessing.LabelEncoder.html`). 

https://github.com/OmicsML/CellPLM/blob/b59ee7688b1bd2a745856a610248b46f15019a22/CellPLM/utils/data.py#L70

The labels can be transformed back with `.inverse_transform` via the label encoder.
